source/indexes/atlas-search-index.txt

```diff
@@ -0,0 +1,274 @@
+
+To create multiple Atlas Search indexes, call the ``create_many()`` method on a ``mongocxx::search_index_view`` instance
+and pass in a vector of ``mongocxx::search_index_model`` instances specifying the Atlas Search indexes you want to create. 
+
+The following example shows how to create multiple Atlas Search indexes:
+
+.. io-code-block::
+   :copyable: true
+
+   .. input:: /includes/indexes/indexes.cpp
+      :start-after: start-create-multiple-search-indexes
+      :end-before: end-create-multiple-search-indexes
+      :language: cpp
+      :dedent:
+   
+   .. output:: 
+      :language: cli
+      :visible: false
+      
+      New index names:
+      myDynamicIndex
+      myStaticIndex
+
+.. _cpp-atlas-search-index-list:
+
+List Search Indexes
+-------------------
+
+To list the Atlas Search indexes on a collection, call the ``list()`` method on a ``mongocxx::search_index_view`` instance. This returns a ``mongocxx::cursor`` instance that you can
+use to iterate over Search indexes. 
+
+The following example prints a list of Search indexes by iterating over a ``cursor::iterator`` 
+that points to the Search indexes from :ref:`Create Multiple Atlas Search Indexes <cpp-atlas-search-index-create-multiple>`: 
+
+.. io-code-block::
+   :copyable: true
+
+   .. input:: /includes/indexes/indexes.cpp
+      :start-after: start-list-search-indexes
+      :end-before: end-list-search-indexes
+      :language: cpp
+      :dedent:
+
+   .. output:: 
+      :language: cli
+      :visible: false
+
+      { "id" : ..., "name" : "myDynamicIndex", "type" : "search", ...,
+        "latestDefinition" : { "mappings" : { "dynamic" : true } }, ...} 
+
+      { "id" : ..., "name" : "myStaticIndex", "type" : "search", ...,
+        "latestDefinition" : { "mappings" : { "dynamic" : false, "fields" : { "title" : { "type" : "string", "analyzer" : "lucene.standard" }, "year" : { "type" : "number" } } } }, ...} 
+
+Alternatively, you can list a specific Search index by passing an index name into the ``list()`` method. 
+This returns a ``mongocxx::cursor`` that points to a result set containing only the specified index. 
+
+The following example uses the ``list()`` method to print an index with the name ``myDynamicIndex``:
+
+.. io-code-block::
+   :copyable: true
+
+   .. input:: /includes/indexes/indexes.cpp
+      :start-after: start-list-search-index
+      :end-before: end-list-search-index
+      :language: cpp
+      :dedent:
+
+   .. output:: 
+      :language: cli
+      :visible: false
+      
+      { "id" : ..., "name" : "myDynamicIndex", "type" : "search", ...,
+        "latestDefinition" : { "mappings" : { "dynamic" : true } }, ...} 
+
+.. _cpp-atlas-search-index-update:
+
+Update a Search Index
+---------------------
+
+To update an Atlas Search index, call the ``update_one()`` method on a ``mongocxx::search_index_view`` instance and pass in the name of the search index 
+you want to update and the definition to update the search index to.
```

Comment:
```suggestion
that you want to update and the definition to update the index to.
```

Reply:
+ Applied to other instances of "you want to" without "that"